pkg/common/common.go

```diff
@@ -26,11 +28,20 @@ const (
 	IMPORTER_DATA_DIR            = "/data"
 	IMPORTER_S3_HOST             = "s3.amazonaws.com"
 	IMPORTER_DEFAULT_PULL_POLICY = string(v1.PullIfNotPresent)
+	CLONER_DEFAULT_PULL_POLICY = string(v1.PullIfNotPresent)
 	// env var names
 	IMPORTER_PULL_POLICY   = "IMPORTER_PULL_POLICY"
 	IMPORTER_ENDPOINT      = "IMPORTER_ENDPOINT"
 	IMPORTER_ACCESS_KEY_ID = "IMPORTER_ACCESS_KEY_ID"
 	IMPORTER_SECRET_KEY    = "IMPORTER_SECRET_KEY"
+	CLONER_PULL_POLICY     = "CLONE_PULL_POLICY"
+	
+	CLONER_SOURCE_PODNAME =          "clone-source-pod"
+	CLONER_TARGET_PODNAME =          "clone-target-pod"
```

Comment:
Will these work if we have multiple clone operations happening at the same time?

Reply:
Will check...

Reply:
so this has to be synchronous because one PVC cannot create the same unix socket (pipe) while another is writing or reading from it. right? so In addition to that, every time when target PVC completes, I need to remove the pipe, so the next source cloning pod can create a new one....
Do you you think we can do cloning in parallel?

Reply:
I will change the script to echo something in loop, just for testing to see if all containers can run in parallel, in order to isolate the issue

Reply:
Is it possible that the pending pods are waiting for the socket mount to complete?

Reply:
Every set of pods (source and target)  have different pod labels and hostpath now.